NativeApp/src/styles/layout.js

```diff
@@ -1,6 +1,7 @@
+import { WHITE } from '../styles/colors';
+
 export const container = {
-  flex: 1,
+  flexGrow: 1,
```

Comment:
Not 100% sure how well this would work when set on `<View />`. It's okay for `<ScrollView />`. Maybe just double check.